migrations/sqlite/2019-03-05-082846_add_fqn/down.sql

```diff
@@ -0,0 +1,3 @@
+-- This file should undo anything in `up.sql`
+ALTER TABLE blogs DROP COLUMN fqn;
+ALTER TABLE users DROP COLUMN fqn;
```

Comment:
sqlite doesn't allow to drop column, you have to do the annoying copy thing

Reply:
Ah, yes, forgot about that… :expressionless: